src/components/MainMenu/messages.js

```diff
@@ -0,0 +1,19 @@
+import { defineMessages } from 'react-intl';
+
+export default defineMessages({
+  new_melding: {
+    id: 'menu.new_melding',
+    description: 'Top navigation new melding label',
+    defaultMessage: 'Nieuwe melding', // TODO decide if we want to use functionality (overlap in functionality)
```

Comment:
It's good practice to have default values in case someone forgets to add a translation string to the json file. If you think this is an issue that should be addressed then please create a ticket instead of a mention in the code here.

Reply:
I don't think we want the default values. Because we also have the same value in the messages for the default language. So there is some duplication there.

Anyway, the react boilerplate extract messages command doesn't work if we leave this one out. So let's keep it in.

BTW, the todo was there so we would forget discussing it in this PR. I'll remove it.